messgen/go_generator.py

```diff
@@ -2,8 +2,23 @@
 
 from messgen import MessgenException
 
+import re
+from pathlib import Path
+from string import Template
+
+enumSuffix = "Enum"
+
+autogenPreamble = [
+        "// Code generated by messgen. DO NOT EDIT.",
+        "",
+]
+
+defaultStringer = Template('fmt.Sprintf("%v", $varName)')
 
 def to_camelcase(str):
+    if not any(c in "_" for c in str):
```

Comment:
Why need this? Microoptimization?

Reply:
it is to prevent something like `AtlasStatus` to be converted to `Atlasstatus`

Reply:
not my call I guess. My goal was to make it working without additional requirements

Reply:
OK